pyiron_atomistics/atomistics/structure/analyse.py

```diff
@@ -26,6 +28,33 @@
 s = Settings()
 
 
+def get_mean_positions(positions, cell, pbc, labels=None):
```

Comment:
So I made it public intentionally, because whenever you have a routine of cluster -> get cluster centres, you'll need this function. For example, imagine a system with precipitates distributed in a box. If you want to have the average position of each precipitate you can label the atoms accordingly and apply this function. This being said, maybe it's not a particularly good idea to make it neither private nor a member function of any class.

Reply:
Ok, I think I understand better. It seems like something that might nicely live directly on analyze. Like 'strict.analyze.get_mean_positions(labels=my_id_subset)'

With that in mind what I would really want is an option to weight by mass so we can readily get com

Reply:
I'll leave com for now because I don't have a good test case and adding com will not change the current functionality anyway.